CMakeLists.txt

```diff
@@ -365,6 +365,7 @@ check_function_exists(vfork HAVE_VFORK)
 check_function_exists(backtrace_symbols HAVE_BACKTRACE_SYMBOLS)
 check_function_exists(pipe2 HAVE_PIPE2)
 check_function_exists(nice HAVE_NICE)
+check_function_exists(malloc_info HAVE_MALLOC_INFO)
```

Comment:
It finds the function:

```
#18 1.399 -- Looking for malloc_info
#18 1.467 -- Looking for malloc_info - found
```

Yet, I get this error:

```
{"error":501,"status":"malloc_info(3) not available."}
```

This line only sets the variable in CMake, it additionally has to be exposed to the C++:
https://github.com/Icinga/icinga2/blob/c20666ae76584656340a33a6036d7c30561dad07/config.h.cmake#L4-L12